packages/spec/integration/jest-preset/__tests__/setup/global-css.js

```diff
@@ -1,2 +1,2 @@
-// eslint-disable-next-line import/no-unresolved
+// eslint-disable-next-line import/no-unresolved, node/no-missing-import
```

Comment:
The filename is misspelled, it's missing a `t`.

Also, when we install [eslint-import-resolver-webpack](npm.im/eslint-import-resolver-webpack), the linter knows how to deal with resource queries.

Reply:
That plugin relies on a webpack config.

Reply:
It can work with defaults, when setting…

```yaml
settings:
  import/resolver: webpack
```

This fixed this specific issue & didn't produce any new ones. I don't know, though, whether this will cause any trouble in the future…